snuba/consumer.py

```diff
@@ -86,27 +78,31 @@ def delivery_callback(self, error, message):
     def flush_batch(self, batch: Sequence[ProcessedMessage]):
         """First write out all new INSERTs as a single batch, then reproduce any
         event replacements such as deletions, merges and unmerges."""
-        inserts = []
-        replacements = []
+        inserts: MutableSequence[WriterTableRow] = []
```

Comment:
It's also very likely that `WriterTableRow` gets replaced by some data structure (`NamedTuple` or `TypedDict`) that can be type checked in the processor as well but that's a ways away.